dev/tidb-in-kubernetes/reference/configuration/storage-class.md

```diff
@@ -83,6 +100,111 @@ For more information, refer to [Kubernetes local storage](https://kubernetes.io/
 
 Refer to [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) for more information on local PV in Kubernetes.
 
+## Disk mount examples
+
+If the components such as monitoring, TiDB Binlog, and `tidb-backup` use local disk to store data, you can mount them on a SAS disk and create separate `StorageClass` for use. For example:
+
+- For a disk storing monitoring data, you can [bind mount](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md#sharing-a-disk-filesystem-by-multiple-filesystem-pvs) them into `/mnt/disks` directory, and create `local-storage` `StorageClass` for them.
```

Comment:
```suggestion
- For a disk storing monitoring data, you can [bind mount](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md#sharing-a-disk-filesystem-by-multiple-filesystem-pvs) it to `/mnt/disks` directory, and create `local-storage` `StorageClass` for it.
```
Bind mount *the disk* to the directory, so use "it".

Reply:
Please check https://github.com/pingcap/docs-cn/pull/2035 for clearer description